config/initializers/strong_migrations.rb

```diff
@@ -0,0 +1,24 @@
+# id columns are generated as bigint, and we sometimes use constraint-less integer columns
+# that reference another table. To ensure we use bigint for those columns, this check raises
+# if a column is added that uses :integer and the column_name ends with "_id".
+#
+# Some "_id" columns may not be references to other tables and may not need to be bigint.
+# To exclude a table/column from this check add a `[table, column]` item into excluded_columns:
+#
+# Ex: excluded_columns = [[:users, :my_new_id]]
+
+excluded_columns = []
+
+StrongMigrations.add_check do |method, args|
+  table, column, type, _options = args
```

Comment:
You should be able to use the parens syntax to extract this:
```suggestion
StrongMigrations.add_check do |method, (table, column, type, _options)|
```

Reply:
Yup, confirmed:

```ruby
(proc { |a, (b, c, *d), e| { a: a, b: b, c: c, d: d, e: e } }).call(1, [2, 3, 4, 5, 6], 7)
=> {:a=>1, :b=>2, :c=>3, :d=>[4, 5, 6], :e=>7}
```